Header header

# x
float32 posX 
float32 rawPosX
float32 velX
float32 rawVelX
float32 crrPosX1
float32 crrVelX1
float32 crrPosX2
float32 crrVelX2
float32 crrXBias
float32 emptyX1
float32 emptyX2


# y
float32 posY 
float32 rawPosY
float32 velY
float32 rawVelY
float32 crrPosY1
float32 crrVelY1
float32 crrPosY2
float32 crrVelY2
float32 crrYBias
float32 emptyY1
float32 emptyY2


# z
float32 posZ 
float32 rawPosZ
float32 velZ
float32 rawVelZ
float32 crrPosZ1
float32 crrVelZ1
float32 crrPosZ2
float32 crrVelZ2
float32 crrZBias
float32 emptyZ1
float32 emptyZ2


# psi (rotation arond z axis)
float32 psi 
float32 rawPsi
float32 velPsi
float32 rawVelPsi
float32 emptyPsi1
float32 emptyPsi2

# debug
float32 crrPosDebug1
float32 crrVelDebug1
#float32 crrBiasDebug1
float32 crrPosDebug2
float32 crrVelDebug2
#float32 crrBiasDebug2